i18n/en-US.properties

```diff
@@ -380,70 +368,34 @@ be.taskDueDate = Due
 be.taskEditErrorMessage = There was an error updating this task.
 # Reject option for a task
 be.taskReject = Decline
-# Error message when rejecting a task fails
-be.taskRejectErrorMessage = An error has occurred while rejecting this task. Please refresh the page and try again.
 # label for button that opens task popup
 be.tasks.addTask = Add Task
-# label for menu item that opens approval task popup
-be.tasks.addTask.approval = Approval
-# description for menu item that opens approval task popup
-be.tasks.addTask.approval.description = Request an approval to move work forward
-# label for menu item that opens general task popup
-be.tasks.addTask.general = Task
-# description for menu item that opens general task popup
-be.tasks.addTask.general.description = Keep track of work that needs to get done
```

Comment:
Why are we deleting tasks messages?

Reply:
I tried rebasing and yarn-run-build with no success. Do you have suggestions?

Reply:
Are you pushing with `--no-verify`?

Reply:
Sometimes, I use --no-verify for incremental changes, but I immediately trigger `push` with it omitted and let it run in the background. This shortens my dev-cycles and allows me to get feedback from my team more quickly. 

Do you think it's breaking i18n?

Reply:
It seems likely, yes. The pre-push script runs other scripts necessary to update the messages, as well as linting and unit tests. We strongly recommend against pushing with `--no-verify` for that reason.